tools/internal_ci/linux/grpc_bazel_build_in_docker.sh

```diff
@@ -25,3 +25,4 @@ git clone /var/local/jenkins/grpc /var/local/git/grpc
 ${name}')
 cd /var/local/git/grpc
 bazel build --spawn_strategy=standalone --genrule_strategy=standalone :all test/... examples/...
+bazel build --spawn_strategy=standalone --genrule_strategy=standalone --define=GRPC_USE_ABSL=1 :grpc
```

Comment:
Adding `--define=GRPC_USE_ABSL=1` here is dirty.  If we want to use absl for bazel builds, the settings should be done somewhere in BUILD files or bazel rc file. Not as a cmdline flag in only one of our testing scripts (there's several other places where we invoke bazel).

Reply:
I added the comment to the script saying this is temporary. This test will be here until absl is fully integrated and make sure that gRPC can work with abseil.